### PR TITLE
Add credentials when calling bootstrapCrossSigning in Cypress tests

### PR DESCRIPTION
**Pull request by devonh**
Dendrite doesn't use the auth shortcut that Synapse uses whereby recently logged in users don't have to re-login when uploading keys. This updates the cypress tests to pass through credentials which are redundant when running with Synapse but required when running with Dendrite.

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

**Review by weeman1337 — COMMENTED** [review]
Tests are passing :+1: Some small comments in the code. Can you have a look?

File: cypress/e2e/crypto/decryption-failure.spec.ts [hunk 1]
```diff
@@ -110,7 +110,14 @@ describe("Decryption Failure Bar", () => {
                     otherDevice = cli;
                     await otherDevice.bootstrapCrossSigning({
                         authUploadDeviceSigningKeys: async (makeRequest) => {
-                            await makeRequest({});
+                            await makeRequest({
```
[the remark this answers is not in the document]
Comment (devonh) [reply cypress/e2e/crypto/decryption-failure.spec.ts]: We can actually avoid this call to bootstrapCrossSigning entirely by providing the `bootstrapCrossSigning = true` option to `loginBot`. Good call.

**Comment by devonh** [comment]
Thanks for the quick review! I've implemented your suggestions and they clean things up a good amount.

**Review by weeman1337 — APPROVED** [review]
LGTM :+1:

**Pull request merged**